plugins/executor/python/python-executor-plugin-configmap.yaml

```diff
@@ -0,0 +1,117 @@
+# This is an auto-generated file. DO NOT EDIT
```

Comment:
Perhaps I missed this somewhere but have we documented how to auto-generate this?

Reply:
`argo executor-plugin build .`